rbi/sorbet/t.rbi

```diff
@@ -286,7 +286,7 @@ module T::Helpers
   # given type of module.
   #
   # For more information, see https://sorbet.org/docs/requires-ancestor
-  sig { params(block: T.proc.returns(Module)).void }
+  sig { params(block: T.proc.void).void }
```

Comment:
@Morriar and I were having a hard time typing this proc so that it accepted `T.class_of` as a return. Let me know if there's a better way to do this.

Reply:
I think given that there's no runtime behavior involved in `requires_ancestor`, we don't need to be too specific about the types here; the primary purpose is communicating information to the static checking phase.  (It would, of course, be nice if there was runtime checking, but that would be a separate discussion to have.)